### DISCUSSION TITLE
Running on windows problems

### DISCUSSION THREAD

**Discussion by TimJMartin** [observablehq/framework#994] [General]
Hi ObservableHQ Team

I saw the new release which now supports windows and keen to give it a spin. 

Using the 'Getting Started Guide' I run the 

```
npm init @observablehq
```

And it seems to just create a standard package.json file rather than initiating the helper script (observable create), giving me the chance to set it all up and get it running. 

I then tried doing a normal npm install and attempting to run npm run create and that errors.

Sorry for such a basic question but if you could let me know the method for windows that would be amazing.

With thanks

Using Node version 20.9.0

**Comment by mbostock** [comment]
What is your Windows environment? Are you using PowerShell? You might try running `npm init "@observablehq"` instead per #986. If that doesn’t work, what version of `@observablehq/create` is it running? Can you show the complete output? And have you tried clearing your `_npx` cache?

**Comment by TimJMartin** [comment]
@mbostock so simple, just wrapping it in quotes was enough. 

I did check the issues but didn't see the Pull request with that info. 

Many thanks and cannot wait to dig into framework
